build/odimra/fabrics.service

```diff
@@ -18,7 +18,7 @@ After=network-online.target
 [Service]
 Type=forking
 Restart=always
-ExecStart=/bin/sh -c 'cd /bin ; export CONFIG_FILE_PATH=/etc/odimra_config/odimra_config.json; sudo -E -u odimra nohup ./svc-fabrics --registry=etcd --framework=GRPC --registry_address=etcd:2379 --server_address=odimra:45106 --client_request_timeout=`expr $(cat $CONFIG_FILE_PATH | grep SouthBoundRequestTimeoutInSecs | cut -d : -f2 | cut -d , -f1 | tr -d " ")`s >> /var/log/odimra_logs/fabric.log 2>&1 &'
+ExecStart=/bin/sh -c 'cd /bin ; export CONFIG_FILE_PATH=/etc/odimra_config/odimra_config.json; sudo -E -u odimra nohup ./svc-fabrics --registry=etcd =GRPC --registry_address=etcd:2379 --server_address=odimra:45106 --client_request_timeout=`expr $(cat $CONFIG_FILE_PATH | grep SouthBoundRequestTimeoutInSecs | cut -d : -f2 | cut -d , -f1 | tr -d " ")`s >> /var/log/odimra_logs/fabric.log 2>&1 &'
```

Comment:
--framework=GRPC is not completely removed in every file from here, please update it in all

Reply:
updated